config/sync/field.field.eventinstance.default.field_branch.yml

```diff
@@ -11,7 +11,7 @@ field_name: field_branch
 entity_type: eventinstance
 bundle: default
 label: Branch
-description: ''
+description: 'When connecting a branch, it will also overwrite the event address with the address from the branch.'
```

Comment:
Please reconsider these descriptions.

I think they are leftovers from the previous implementation where the logic was reversed.

This also applies for the three other descriptions.